### ISSUE TITLE
How to use?

### ISSUE THREAD

**Issue by emcniece** [joslarson/maschine-bitwig#11]
Hi! The `scripts{}` section of `package.json` is empty, so I'm not sure exactly how to build & run this. Executing `./node_modules/.bin/webpack` creates `dist/` and the associated files as defined in `webpack.config.js` but Node can't run the generated JS.

Is this supposed to be run from a browser?

**Comment by joslarson** [comment]
@emcniece Hello probably 1.5 years too late. Ever since I started working somewhere for my day job that uses GitHub, the notification system has become useless (or maybe I just need to learn how to use it better).

To answer the second part of your question, this is not meant to run in NodeJS or the browser, but in Bitwig, which has a JS API. I built a Webpack plugin that bundles modules in a way that's compatible with Bitwig's env, and that allows me to use JS modules and to bundle code from npm (which wouldn't work in Bitwig's env otherwise).

And to answer the first part about how to build/run it, I never quite got this finished in a way to easily share it, but eventually the idea is to have releases here and make a single js file available to download per release per device type. I've recently started working on this again though.
